### PR TITLE
Add Ansible and use it to install New Relic Infrastructure

### PR DESCRIPTION
**Pull request by giorgiosironi**
Depends on https://github.com/libero/infrastructure/pull/31

Ansible playbooks are applied on instance creation, but can also be re-applied to existing instances so that we don't have to recreate a server from scratch to add a package.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Applied playbooks to `unstable` and tested everything from scratch on a new `test` environment, now destroyed.

**Comment by giorgiosironi** [comment]
Turns out the `registration.key` needs to be per-environment; otherwise a `terraform destroy` on an environment invalidates it for usage on other environments, violating their isolation.

**Comment by giorgiosironi** [comment]
I guess some manual taint and re-creation will be necessary to solve the `registration.key` issues...

File: playbooks/boot.playbook [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+- hosts: '*'
+  gather_facts: False
+  remote_user: ubuntu
+  become: yes
```
Comment (GiancarloFusiello) [inline playbooks/boot.playbook]: interesting key name...

**Pull request merged**